okhttp-tests/src/test/java/okhttp3/EventListenerTest.java

```diff
@@ -611,88 +613,175 @@ private void enableTlsWithTunnel(boolean tunnelProxy) {
     server.useHttps(sslClient.socketFactory, tunnelProxy);
   }
 
-  static final class DnsStart {
+  static class CallEvent {
```

Comment:
Hmm, not sure what I think of this. On the one hand it cuts down on the boilerplate and redundancy, but on the other hand we lose some type information on the params.